src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  background-color: black;
+}
+
+p {
+  margin: 0;
+}
+
+.inside_block {
+  background-color: #ddd;
+}
+
+.midle_block {
+  background-color: #ffa500;
+}
+
+.outside_block {
+  width: 350px;
+  box-sizing: border-box;
+  background-color: #308dc0;
+  margin: 20px 0 40px 12px;
```

better to add body `margin: 0`, then:
```suggestion
  margin: 20px 0 40px 20px;
```